BusBoard.ConsoleApp/CliInterface.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusBoard.ConsoleApp
+{
+    class CliInterface
+    {
+        public static void RunUserCommandLoop(TflApi tflApi)
+        {
+            while (true)
+            {
+                Console.WriteLine("\n\"[Bus stop code]\" to list arrivals at that stop");
+                Console.WriteLine("\"Quit\" to quit");
+
+                Console.WriteLine("Please type an option: ");
+
+                var input = Console.ReadLine();
+
+                if (input == "Quit")
+                {
+                    break;
+                }
+
+                try
+                {
+                    var predictions = tflApi.GetSortedArrivals(input);
+
+                    foreach (var prediction in predictions.Take(3))
```

Comment:
Could do a similar approach here to FizzBuzz where you join up all the strings and print them together, slightly reduces nesting. But up to you, this is fine as well 🙂

Reply:
I'm not quite sure what you mean here. I suppose you could use something like a `Select` to extract the prediction summaries, and join them together using `String.Join`? I needed to refactor this out into its own function for part 2, so I brought that forward into 6b680cb, but can still refactor it like that if you think it will be more readable?

Reply:
Yes, that's exactly what I meant 🙂  I'd probably have a slight personal preference for that as it reduces the nesting, and I find nesting really annoying when reading code. But it's a pretty marginal difference between the two, I can see the argument for going with this solution (avoids a select + join), so feel free to leave it as it is if you prefer 🙂

Reply:
Ah and I see now what you mean about the refactoring. With it in its own method I guess it's just that select + join would be more functional. I would probably still prefer select + join, but your stringbuilder solution is fine as well, so up to you 🙂

Reply:
Yeah I've just done that in 95cc28e, I just thought I'd check in case I was missing something obvious